projects/isolate_proto/src/isolate_proto/controller.proto

```diff
@@ -379,8 +379,9 @@ message RunnerInfo {
 
     enum State {
         RUNNING = 0;
-
         PENDING = 1;
+        SCHEDULING = 2;
+        SETUP = 3;
```

Comment:
I think we can reduce this to


```suggestion
        PENDING = 1;
        SETUP = 2;
```

what we pring in the CLI outpout can be `scheduling` but instead of making `PENDING` obsolete, we reuse it.